inlong-sort/sort-connectors/iceberg-dlc/pom.xml

```diff
@@ -88,8 +88,9 @@
         </dependency>
 
         <dependency>
-            <groupId>org.apache.iceberg</groupId>
-            <artifactId>iceberg-flink-runtime-1.13</artifactId>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>sort-connector-iceberg</artifactId>
+            <version>1.3.0-SNAPSHOT</version>
```

Comment:
Please use inlong-version to replace it.

Reply:
will do it in another PR with other components